ledger-tool/src/blockstore.rs

```diff
@@ -332,7 +332,7 @@ pub fn blockstore_subcommands<'a, 'b>(hidden: bool) -> Vec<App<'a, 'b>> {
                     .long("target-db")
                     .value_name("DIR")
                     .takes_value(true)
-                    .help("Target db"),
+                    .help("Directory to write inner \"rocksdb\" in."),
```

Comment:
You beat me to creating a PR, but I think we should change the argument name to `--target-ledger`. This would make it more consistent with the existing `--ledger` argument

Reply:
307a2013b6 - something like this?